<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>windowQTClass</class>
 <widget class="QMainWindow" name="windowQTClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>629</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>windowQT</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="exit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>100</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>exit</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="containerConnectionStatus">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>0</y>
      <width>120</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Connection Status</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="serverStatus">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Server</string>
     </property>
    </widget>
    <widget class="QFrame" name="serverStatusIndicator">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>18</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: black;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QLabel" name="dllStatus">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>DLL</string>
     </property>
    </widget>
    <widget class="QFrame" name="dllStatusIndicator">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: black;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="containerInjector">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>261</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Injector</string>
    </property>
    <widget class="QComboBox" name="processPicker">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="modulePicker">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="injectButton">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>inject</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="containerDeinject">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>170</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Deinject</string>
    </property>
    <widget class="QPushButton" name="deinjectButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>deinject</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>629</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="windowQT.qrc"/>
 </resources>
 <connections/>
</ui>
